src/CarbonAware.NewCLI/CommandKeywords/Emissions/EmissionsListCommand.cs

```diff
@@ -0,0 +1,79 @@
+﻿using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Model;
+using System.CommandLine;
+using System.Text.Json;
+
+namespace CarbonAware.NewCLI.CommandKeywords.Emissions;
+
+/// <summary>
+/// Defines the command to help get a list of emission data by location for a specified time period.
+/// </summary>
+/// <option name="location"> String named location.</option>
+/// <option name="time"> [Optional] Start time for the data query.</option>
+/// <option name="toTime"> [Optional] End time for the data query.</option>
+/// <returns>Array of EmissionsData objects that contains the location, time and the rating in g/kWh</returns>
+public static class EmissionsListCommand
+{
+    static ICarbonAwareAggregator? _aggregator;
+    public static void AddEmissionsListCommand(this Command parent, ICarbonAwareAggregator aggregator)
+    {
+        _aggregator = aggregator;
+
+        var listCommand = new Command("list", "Lists emission data for given locations and times.");
+        
+        var locationOption = new Option<string[]>("--locations")
+        {
+            Description = "Space separated strings of Locations to get emissions data for.",
+            IsRequired = true,
+            AllowMultipleArgumentsPerToken = true,
+            Arity = ArgumentArity.OneOrMore
+        };
+
+        var startTimeOption = new Option<DateTime?>("--startTime", getDefaultValue: () => null)
+        {
+            Description = "Start time for emissions data window",
+            IsRequired = false
+        };
+
+        var toTimeOption = new Option<DateTime?>("--toTime", getDefaultValue: () => null)

```

Comment:
`toTime` seems a bit strange and very specific to the param names form the controller. Maybe `endTime` is clearer